APMrover2/defines.h

```diff
@@ -62,7 +62,7 @@ enum mode {
 #define MASK_LOG_PM             (1<<3)
 #define MASK_LOG_CTUN           (1<<4)
 #define MASK_LOG_NTUN           (1<<5)
-#define MASK_LOG_MODE           (1<<6)
+//#define MASK_LOG_MODE         (1<<6) // depreciated
```

Comment:
`deprecated`!

Further, your changes here make it more than deprecated - it's been *eliminated*, so perhaps `// no longer used` would be more appropriate?